tests/c21client/test_client.py

```diff
@@ -1,29 +1,80 @@
+import os
 import pytest
+import requests
 from c21client.client import Client
 
-
 BASE_URL = "http://localhost:8080"
 
 
+@pytest.fixture
+def setup():
+    os.remove('client.cfg')
```

Comment:
What happens if you run the tests after the actual client had run once? This will delete a valid `client.cfg` file. We should probably create a `mock_client.cfg` file in the tests/c21client directory when running this test. @francisseverino

Reply:
I can mock the methods that read and write to the file. This will lower coverage, but as you mentioned, I too think that it is a necessary change.

Reply:
I'm going to merge this pull request before it gets any larger, but you should make another pull request when you complete that feature.